bot/controllers/invite.js

```diff
@@ -28,12 +28,20 @@ class inviteController extends BaseController {
   // this message will be sent to the user's dm with their username
   inviteAction() {
     const { message } = this;
-    const validDomains = ['student.fullsail.edu', 'fullsail.edu', 'fullsail.com'];
     const email = message.parsed[1].toLowerCase();
+    const invitePointsAwarded = 1;
     const emailDomain = email.split('@').pop();
+    const filter = /^([a-zA-Z0-9_.-])+@(([a-zA-Z0-9-])+.)+([a-zA-Z0-9]{2,4})+$/;
+
     // create the invite for the channel
     message.channel.createInvite().then((invite, err) => {
       if (err)util.log('create invite err : ', err, 3);
+      // Test to see if the users email exist
+      if (!filter.test(email.value)) {
+        // Send user message if the email is not correct
```

Comment:
If the email is not valid, will the rest of the code still run? From what it looks like to me it will send the message all while the code below still tries to run. If that's the case, try adding an else conditional so that the code only runs if the email is valid.